### PR TITLE
Add comparison of several thresholding methods to testing.

### PR DESCRIPTION
**Pull request by HGeerlings**
Can’t quite get subplots to display, however interesting that the Yen thresholding returns the same thresholding values as our TRACR implementation.

### CODE REVIEW

**Comment by bkappes** [comment]
@HGeerlings I downloaded the branch, but can't seem to get it to run. I'm getting errors across `test.test_otsu.TestClass.test_tif_series_*` because the *.tif files are not included in the repo. Where can I get those?

**Comment by bkappes** [comment]
I think the goal of this PR is to test our Otsu implementation against the skimage implementation? If so, can you add a quantitative metric, i.e. some acceptable percentage difference between the two?

Here's what I'm thinking:
1. Use a stock image, like `scipy.misc.face` and threshold with the sklearn Otsu (`imageA = image > sklearn.filter.otsu_threshold(image)`) and our Otsu (`imageB = image > tracr.actions.threshold.otsu(image)[0]`).
2. Perform a visual inspection of `imageA` and `imageB`. Heuristically, do they look good?
3. Quantify the thresholded image, e.g. `sumA = np.sum(imageA)` and `sumB = np.sum(imageB)`. How different are these sums, e.g. `comp = 2*float(sumA - sumB)/(sumA + sumB)`? If the thresholded images from step 2 look good, then use `comp` as a basis for testing. I.e. let's say comp = 0.024358604, then add the following to your test:

```
assert comp < 0.025, 'Inconsistent sklearn and tracr Otsu implementations'
```

You can then disable the visualization of `imageA` and `imageB` -- we have a numerical and computer-testable metric for validating the Otsu implementation.

File: .gitignore [hunk 1]
```diff
@@ -98,3 +98,7 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Mac OS X stuff
+Thumbs.db
+.DS_Store
```
Comment (sinistral) [inline .gitignore:104]: Would you mind removing the existing .DS_Store files that have already been added, too, please?  

`find . -name ".DS_Store" -exec git rm {} \;`
